src/main/webapp/app/shared/pipes/sanitize-html.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import * as DOMPurify from 'dompurify';
+
+@Pipe({
+    name: 'sanitizeHtml',
+})
+export class SanitizeHtmlPipe implements PipeTransform {
+    transform(html: string): string | null {
+        return html ? DOMPurify.sanitize(html) : null;
```

Comment:
Angulars sanitizer messes up the katex styling, so I used DOMPurify (https://github.com/cure53/DOMPurify).